apps/omg_watcher/lib/omg_watcher/supervisor.ex

```diff
@@ -22,33 +22,58 @@ defmodule OMG.Watcher.Supervisor do
 
   alias OMG.Watcher
 
+  if Application.get_env(:omg_watcher, :sql_sandbox) do
+    defmodule Sandbox do
```

Comment:
I can understand that we need to inject this mode change just after repo is alive. However I thought error appears only when mode is changed during [this](https://github.com/omisego/elixir-omg/blob/ff0c73cc4b08e7971005b2192c4cf44ac1ea1bcd/apps/omg_watcher/lib/omg_watcher/block_getter.ex#L63) block_getter's call. 
I think can think a little more to avoid this huge changes which are only here to kill some annoying log...